### DISCUSSION TITLE
value of [role="textbox"]

### DISCUSSION THREAD

**Discussion by eps1lon** [eps1lon/dom-accessibility-api#4] [accname]
I couldn't find any document that defines how the value for those elements is computed. Most examples also specify `contenteditable` (is this required?). I'm currently assuming that the value of `*[role="textbox"]` is its `textContent`.

See #3

**Comment by sjsamphex** [comment]
Should we be checking the label of the textbox in this case? For example, assistive technology would read the label of the textbox example in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input as "Name (4 to 8 characters):"

**Reply by eps1lon** [reply]
Isn't this computing the name? This topic is about the value of the textbox e.g. how would you write a `<div role="textbox" />` as `<input value="??" />`.

**Reply by sjsamphex** [reply]
Ah I misunderstood - thanks for the clarification.
